R/zchunk_L210.Resources_china.R

```diff
@@ -0,0 +1,168 @@
+    L1231.out_EJ_province_elec_F_tech <- get_data(all_data, "temp-data-inject/L1231.out_EJ_province_elec_F_tech")
+    L210.RenewRsrc <- get_data(all_data, "L210.RenewRsrc")
+    L210.UnlimitRsrc <- get_data(all_data, "L210.UnlimitRsrc")
+    L210.UnlimitRsrcPrice <- get_data(all_data, "L210.UnlimitRsrcPrice")
+    L210.SmthRenewRsrcCurves_wind <- get_data(all_data, "L210.SmthRenewRsrcCurves_wind")
+    # ===================================================
+    cement_provinces <- unique( L1321.out_Mt_province_cement_Yh$province )
+
+    no_geo_provinces <- gcamchina.PROVINCES[ gcamchina.PROVINCES %in% L1231.out_EJ_province_elec_F_tech$province[ L1231.out_EJ_province_elec_F_tech$fuel == "geothermal" &
+                                                                                                                     L1231.out_EJ_province_elec_F_tech$X2010 == 0 ] ]
+    no_geo_provinces_resource <- tibble( region = no_geo_provinces, renewresource = "geothermal")
+
+    # L210.RenewRsrc_CHINA: renewable resource info in the provinces
+    L210.RenewRsrc_CHINA <- L210.RenewRsrc %>%
+      filter(region == "China",
+             renewresource %in% gcamchina.PROVINCE_RENEWABLE_RESOURCES) %>%
+      write_to_all_provinces(LEVEL2_DATA_NAMES[["RenewRsrc"]]) %>%
+      # Remove geothermal from provinces that don't have it
+      anti_join(no_geo_provinces_resource, by = c("region", "renewresource")) %>%
+      mutate(market = if_else(renewresource != "onshore wind resource", "China", region))
+
+    # L210.UnlimitRsrc_CHINA: unlimited resource info in the provinces
+    # TODO: If needed, add in capacity factor (from old data system, seems to be 0.3 for solar, 0 for limestone)
+    L210.UnlimitRsrc_CHINA <- L210.UnlimitRsrc %>%
+      filter(region == "China",
+             unlimited.resource %in% gcamchina.PROVINCE_UNLIMITED_RESOURCES) %>%
+      write_to_all_provinces(LEVEL2_DATA_NAMES[["UnlimitRsrc"]])
+
+    L210.UnlimitRsrc_limestone_CHINA <- L210.UnlimitRsrc_CHINA %>%
+      filter(unlimited.resource == "limestone",
+             region %in% cement_provinces)
+
+    L210.UnlimitRsrc_CHINA <- L210.UnlimitRsrc_CHINA %>%
+      filter(unlimited.resource != "limestone")
+
+    # L210.UnlimitRsrcPrice_CHINA: unlimited resource prices in the provinces
+    L210.UnlimitRsrcPrice_CHINA <- L210.UnlimitRsrcPrice %>%
+      filter(region == "China",
+             unlimited.resource %in% gcamchina.PROVINCE_UNLIMITED_RESOURCES) %>%
+      write_to_all_provinces(LEVEL2_DATA_NAMES[["UnlimitRsrcPrice"]])
+
+    L210.UnlimitRsrcPrice_limestone_CHINA <- L210.UnlimitRsrcPrice_CHINA %>%
+      filter(unlimited.resource == "limestone",
+             region %in% cement_provinces)
+
+    L210.UnlimitRsrcPrice_CHINA <- L210.UnlimitRsrcPrice_CHINA %>%
+      filter(unlimited.resource != "limestone")
+
+    # TODO: L210.SmthRenewRsrcTechChange_CHINA (look at old data system code chunk) can be added back in when we have distributed_solar added by province
```

Comment:
Same; feel free to make a `gds-china` issue label @sypnl

Reply:
opened a new issue on provincial vs. national resource curves (todos marked in this chunk). Used the gcam-china issue label created last year.